cts/schemas/test-3/ref/nvpair-hash-default.ref-99

```diff
@@ -59,198 +59,359 @@
          * No rule definitions should be deleted.
       -->
     <crm_config>
+      <cluster_property_set id="pcmk__3_10_upgrade-cluster-properties1-option1-no-hash-default" score="7">
+        <rule id="pcmk__3_10_upgrade-cluster-properties1-option1-no-hash-default-rule">
+          <rule id-ref="cluster-properties1-rule"/>
+          <rule id="pcmk__3_10_upgrade-cluster-properties2-rule-negated" negate="1">
+            <date_expression id-ref="cluster-properties2-rule-expr"/>
```

Comment:
This actually fails to validate because rule expressions (of all the types) can't be id-refs.

I'll need to do one of the following:
* add support for id-ref expressions (which may complicate things like removal via CLI tools)
* add support for `negate` in rule id-refs and not just rule definitions (even though this makes good sense conceptually as "negate the referenced rule," I don't like having other attributes alongside `id-ref`, because it complicates resolution)

Not sure which one is the better idea...